styles/components/_modal.scss

```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
+    font-size: 2.5rem;
+  }
+
+  &__win-image {
+    position: relative;
+    width: 10rem;
+    height: 10rem;
+    border: 2px solid #ffca36;
+    margin: 5rem auto auto;
+    border-radius: 50%;
+  }
+
+  &__win-avatar {
+    position: relative;
+    top: 2rem;
+    width: auto;
+    height: 6rem;
+  }
+
+  &__win-medal {
+    position: absolute;
+    right: -2rem;
+    bottom: -2rem;
+    height: 6rem;
+  }
+
+  &__win-text {
+    margin-top: 2rem;
+    color: #71cfd7;
+    font-size: 2rem;
+  }
+
+  &__win-button {
```

Comment:
it is the same button as on the landing page. Perhaps you could talk it over with @paula0403 and figure out a styling together? It also needs to have the border removed (grey and black) and a separate styling for hover and focus (remove the black outline).

Reply:
For now all settings will be added in this specific class, after Landing Page will be merged I will change it. (added comment about this)

Reply:
👍